vpi/list/vpi.c

```diff
@@ -0,0 +1,71 @@
+#include <vpi_user.h>
```

Comment:
The thing is that to demonstrate very basic usage of VPI, the VHDL component under test do not matter so much.  So, for helloworld I can, for example, just put a complete void top module without any signals, because at the end the VPI code is interacting with the simulator but not with the simulation. If you think it is better I can try to make the ent.vhd and tb.vhd even more concise in each test.

Reply:
This. In VHPIDIRECT examples, many C sources are common but VHDL sources need to be different. Here, as you said, VHDL sources are almost irrelevant (as long as they contain the minimum features that we want to test). Hence, instead of making `ent.vhd` and `tb.vhd` more concise in each test, I would suggest to make them general enough so that they can used in any VPI example (last structure above). If some example in the future requires some weird feature in VHDL, we will consider adding different VHDL sources to the corresponding subdir. But, for now, it seems unnecessary.